platform-configs/msi-pro-z690-a-wifi-ddr4.robot

```diff
@@ -3,17 +3,30 @@ Resource    include/msi-z690-common.robot
 
 
 *** Variables ***
-${FW_VERSION}=                          v1.1.2
+${FW_VERSION}=                          v1.1.4-rc1
 ${DMIDECODE_SERIAL_NUMBER}=             N/A
-${DMIDECODE_FIRMWARE_VERSION}=          Dasharo (coreboot+UEFI) v1.1.3
+${DMIDECODE_FIRMWARE_VERSION}=          Dasharo (coreboot+UEFI) v1.1.4-rc1
 ${DMIDECODE_PRODUCT_NAME}=              MS-7D25
-${DMIDECODE_RELEASE_DATE}=              11/27/2023
+${DMIDECODE_RELEASE_DATE}=              10/07/2024
```

Comment:
@JanPrusinowski Likely we need this to be ported to: https://github.com/Dasharo/open-source-firmware-validation/commits/regression-msi/

Reply:
https://github.com/Dasharo/open-source-firmware-validation/commit/c2aec8eb46036528534b53245960e8f184f8ddca